Cargo.toml

```diff
@@ -19,13 +19,19 @@ log = "0.4.17"
 simple_logger = "4.0.0"
 tokio = { version = "1.25.0", features = ["rt", "rt-multi-thread"] }
 tonic = "0.8.3"
-tonic_lnd = "0.5.1"
+tonic_lnd = { git = "https://github.com/Kixunil/tonic_lnd", rev = "0fad10d91b242920723fedafb41251878344363f" }
 hex = "0.4.3"
 configure_me = "0.4.0"
 bytes = "1.4.0"
 
 [dev-dependencies]
+bitcoincore-rpc = { package="core-rpc", version = "0.17.0" }
+bitcoind = { version = "0.30.0", features = [ "22_0" ] }
```

Comment:
Looks like there's a new version published recently. Will just need to regenerate the lockfile too. Still uses zip 0.5 unfortunately.

```suggestion
bitcoind = { version = "0.31.1", features = [ "24_0_1" ] }

```

Reply:
I see zip was downgraded in the crate for compatibility reasons: https://github.com/RCasatta/bitcoind/commit/e0956143c0d89d0e91c8b481fd3add3c449a6209

I honestly feel that since this is just for testing we can safely ignore for now, but if we want all greens then I've updated it as the compatibility issue won't affect us:

```suggestion
bitcoind = { git = "https://github.com/dunxen/bitcoind", rev = "ce31f17662cc230bbf2cd7c4fee09b75c272bbac", features = [ "24_0_1" ] }
```

Reply:
Cool! Thanks for looking into this. I'm down to use your fork... Though if we need to pull in future updates from bitcoind, maintaining that fork might be a bit annoying. Perhaps a third option would be to ignore this particular vulnerability, with:

`cargo audit --ignore RUSTSEC-2020-0071`

Though I'm not sure if we can pass in this ignore condition into github actions https://github.com/lndk-org/lndk/blob/master/.github/workflows/audit.yml

Reply:
True I think the only way we can achieve ignoring that advisory is with a .audit.toml. At least that will keep it explicitly tracked in the repo. The action doesn't accept any args.

https://docs.rs/crate/cargo-audit/0.10.0/source/audit.toml.example

Reply:
Oh awesome, just pushed up a new commit with that file added and it did the trick. Yay green checkmarks, I bow to thee

Reply:
Haha, forwarding the bow to the wider web :D